### PR TITLE
Add README.rst and remove description from __openerp__.py

### PR DESCRIPTION
**Pull request by Doscaal**

### CODE REVIEW

File: stock_reserve_sale/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+==================
+
+Allows to create stock reservations for quotation lines before the
+confirmation of the quotation.  The reservations might have a validity
+date and in any case they are lifted when the quotation is canceled or
+confirmed.
+
+Reservations can be done only on "make to stock" and stockable products.
+
+The reserved products are substracted from the virtual stock. It means
```
Comment (StefanRijnhart) [inline stock_reserve_sale/README.rst]: typo: subtracted

File: stock_reserve_sale/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+If you try to make a reservation for an order whose lines have different, you
+will get a message suggesting to reserve each line individually. There is no
+module dependency: this modules is fully functional even without ownership
+management.
+
+
+Installation
+============
+
+To install this module, you need to:
```
Comment (StefanRijnhart) [inline stock_reserve_sale/README.rst]: Dependencies are installed automatically. Maybe change to something like 'The installation of this module will also install the following modules:'

File: stock_reserve_sale/README.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Stock Reserve Sale
+==================
+
+Allows to create stock reservations for quotation lines before the
+confirmation of the quotation.  The reservations might have a validity
```
Comment (pedrobaeza) [inline stock_reserve_sale/README.rst]: Remove double space after the dot.

**Comment by Doscaal** [comment]
I fix these points in my previous commit (subtracted, space, installation section)

**Comment by pedrobaeza** [comment]
:+1:

File: stock_reserve_sale/README.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Stock Reserve Sale
+==================
+
+Allows to create stock reservations for quotation lines before the
+confirmation of the quotation.The reservations might have a validity
```
Comment (max3903) [inline stock_reserve_sale/README.rst]: Please add a space after the dot

File: stock_reserve_sale/README.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+date and in any case they are lifted when the quotation is canceled or
```
Comment (max3903) [inline stock_reserve_sale/README.rst:9]: cancelled

File: stock_reserve_sale/README.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+confirmed.
+
+Reservations can be done only on "make to stock" and stockable products.
+
+The reserved products are subtracted from the virtual stock. It means
+that if you reserved a quantity of products which bring the virtual
+stock below the minimum, the orderpoint will be triggered and new
+purchase orders will be generated. It also implies that the max may be
+exceeded if the reservations are canceled.
```
Comment (max3903) [inline stock_reserve_sale/README.rst:18]: cancelled

Comment (Doscaal) [inline stock_reserve_sale/README.rst:18]: Sorry not fully wake up. Fix

Comment (yvaucher) [inline stock_reserve_sale/README.rst:18]: canceled is correct it can be written both ways

**Comment by max3903** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**